packages/material-ui/src/SwipeableDrawer/SwipeableDrawer.js

```diff
@@ -275,6 +273,22 @@ const SwipeableDrawer = React.forwardRef(function SwipeableDrawer(props, ref) {
         maxTranslate,
       );
 
+      if (open) {
+        const paperHit = horizontalSwipe ? currentX < maxTranslate : currentY < maxTranslate;
```

Comment:
to move inside the if branch?

Reply:
I don't understand what you mean.

Reply:
https://github.com/oliviertassinari/material-ui/commit/3b884b7cf85f152604e550ef44fcdf27441757ac